index.html

```diff
@@ -0,0 +1,213 @@
+              <h3>KIA</h3>
+              <p>Responsive Website </p>
+          </div>
+          </div>
+           <!---second project-->
+     
+          <div class=" group  bg-blue-200 relative w-1/4  m-5  overflow-hidden">
+            <img src="./Images/1.jpg" alt="Avatar" class="object-cover w-full h-full opacity-50  transition duration-500 group-hover:opacity-100" />
+            <div class="absolute w-full p-3 bottom-0 inset-x-0 bg-blue-300 text-gray-800 text-lg text-center leading-5 group-hover:opacity-0">
+                <h3>kalleh</h3>
+                <p>Responsive Website </p>
+            </div>
+            </div>
+
+  
+     <!---third project-->
+        <div class=" group  bg-red-200 relative w-1/4  m-5  overflow-hidden">
+          <img src="./Images/3.jpg" alt="Avatar" class="object-cover w-full h-full opacity-50  transition duration-500 group-hover:opacity-100" />
+          <div class="absolute w-full p-3 bottom-0 inset-x-0 bg-red-300 text-gray-800 text-lg text-center leading-5 transition duration-500 group-hover:opacity-0">
+              <h3>Solico</h3>
+              <p>Internal Portal </p>
+          </div>
+          </div>
+          <!---forth project-->
+          <div class=" group  bg-green-200 relative w-1/4  m-5  overflow-hidden">
+            <img src="./Images/4.jpg" alt="Avatar" class="object-cover w-full h-full opacity-50 transition duration-500 group-hover:opacity-100" />
+            <div class="absolute w-full p-3 bottom-0 inset-x-0 bg-green-300 text-gray-800 text-lg text-center leading-5 group-hover:opacity-0">
+                <h3>Bonny Chow</h3>
+                <p>Document Manager System </p>
+            </div>
+            </div>
+                
+                </div>
+            
+            </div>
+      </section> 
+      <section class="container mx-12 mb-12 mt-12 ">
+    
+        <div class=" text-gray-700 mb-12 mt-12 text-center uppercase font-bold text-3xl">lets talk</div>
+          <ul class=" flex flex-row justify-center mt-4 gap-24">
+              <li class="">
+                <a
+                  href="samiraghadirzad@gmail.com"
+                  class=" text-gray-800 "
+                >
+                  <i class="fa fa-envelope  fa-2x" aria-hidden="true"></i>
+                </a>
+              </li>
+              <li>
+                <a
+                  href="https://www.linkedin.com/in/samira-ghadirzad-a5b9688a/"
+                  class="flex items-center justify-center   text-gray-800 "
+                >
+                <i class="fab fa-linkedin-in fa-2x"></i>
+                </a>
+              </li>
+              <li>
+                  <a
+                    href="https://github.com/samizad"
+                    class=" text-gray-800"
+                  >
+                    <i class="fab fa-github fa-2x"></i>
+                  </a>
+                </li>
+                <li>
+                  <a
+                    href=""
```

Comment:
If there's no link to point to, either remove the whole block (in case you don't have a Twitter account) or put a # sign here.
But it's better to not have this if you don't have a Twitter account in this case...